assets/js/components/Dashboard/PaMessaging/NePaMessagePage/WhereCard.tsx

```diff
@@ -0,0 +1,25 @@
+import React, { ComponentType } from "react";
+import { Card, Button } from "react-bootstrap";
+import {
+  NewPaMessagePageState,
+  NewPaMessagePageReducerAction,
+} from "./NewPaMessagePage";
+import { PlusLg } from "react-bootstrap-icons";
+
+interface Props {
+  pageState: NewPaMessagePageState;
+  dispatch: React.Dispatch<NewPaMessagePageReducerAction>;
+}
+
+const WhereCard: ComponentType<Props> = (_props) => {
```

Comment:
**quibble:** I've noticed using `ComponentType` is pretty common but I find it a bit broad and heavy handed. 

`ComponentType` is defined as either a [`ClassComponent` or `FunctionComponent`](
https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b59dc3ac1e2770fbd6cdbb90ba52abe04c168196/types/react/index.d.ts#L104) and while `WhereCard` is most close to a `FunctionComponent`, we aren't setting any of the [static properties](https://github.com/DefinitelyTyped/DefinitelyTyped/blob/b59dc3ac1e2770fbd6cdbb90ba52abe04c168196/types/react/index.d.ts#L1123-L1184) on these functions.

It is just as correct to use a regular TypeScript function argument annotation:

```suggestion
const WhereCard = (_props: Props) => {
```